src/emailLogin/runMigration.js

```diff
@@ -42,13 +42,22 @@ const getSiteAndContributors = async (site, dbClient) => {
     const whitelistQuery = 'SELECT * FROM "whitelist" WHERE expiry IS NULL;';
     const whitelistedDomains = (await dbClient.query(whitelistQuery)).rows.map((whitelistData) => whitelistData.email);
 
+    // Migration army users - only whitelisted gmail/ymail/hotmail accounts in our db
+    // eslint-disable-next-line quotes
+    const isomerWhitelistQuery = `SELECT * FROM "whitelist" WHERE email LIKE '%@gmail.com' OR email LIKE '%@ymail.com' OR email LIKE '%@hotmail.com';`;
+    const additionalIsomerEmails = (await dbClient.query(isomerWhitelistQuery)).rows.map((whitelistData) => whitelistData.email);
+
     const siteMemberValues = [];
     userData.forEach((user) => {
       const userId = user.id;
       if (!user.email) {
         // User not registered to an email
         return;
       }
+      if (additionalIsomerEmails.includes(user.email) || user.email.endsWith('@open.gov.sg')) {
```

Comment:
Hey I am quite confused here :(
While _all_ of our migration army users have whitelisted emails that end with gmail/ymail/hotmail accounts, not _all_ of those emails that end with gmail/ymail/hotmail are part of of our migration army correct? 

As part of the migration, shouldnt we treat migration army as power collaborators rather than admins, and potentially use github as a source of truth to map which repos they have collaborator access to? 

can huddle on this if typing is hard HAHA

Reply:
I've checked with Jackson - the only whitelisted emails which end with generic emails (gmail, ymail, hotmail) are from our migration army, he's always ensured that agencies only send them specialised emails to whitelist.

I'm wondering about the migration army access actually - I decided to remove them so that agencies don't see them as collaborators later on after they're on the identity flow. We can chat more about this offline!

Reply:
oki catch you tomorrow sir Alex~

Reply:
As discussed offline - we're going to only give agency users access via email login, since the majority of the sites will already be okay by the time we provide them with email login access. Any future changes that they want us to change which involves the migration army will involve us providing them with the email of the user to provide access, so that agencies can do their own user management in future